templates/home.html

```diff
@@ -9,77 +9,77 @@ <h1 class="main-logo">kerala<span style="font-weight: normal;">rescue</span></h1
     <p style="margin-bottom:20px;">
         <i>An initiative by Govt. of Kerala, Kerala State IT Mission and IEEE Kerala Section<br>
         For effective collaboration and communications between authorities, volunteers and public </i><br/>
-        <strong>ദുരിതാശ്വാസ പ്രവര്‍ത്തനങ്ങളും രക്ഷാ ദൗത്യങ്ങളും ഏകോപിപ്പിക്കാന്‍</strong><br/>
+        <strong><span class="notranslate">ദുരിതാശ്വാസ പ്രവര്‍ത്തനങ്ങളും രക്ഷാ ദൗത്യങ്ങളും ഏകോപിപ്പിക്കാന്‍</span></strong><br/>
```

Comment:
Please add a comment here as well explaining that `notranslate` is used to restrict google translate.